Userland/Libraries/LibCore/Account.h

```diff
@@ -11,15 +11,20 @@
 #include <AK/Types.h>
 #include <AK/Vector.h>
 #include <pwd.h>
-#ifndef AK_OS_MACOS
+#ifndef AK_OS_BSD_GENERIC
 #    include <shadow.h>
-#else
-#    include <LibC/shadow.h>
 #endif
 #include <sys/types.h>
 
 namespace Core {
 
+#ifdef AK_OS_BSD_GENERIC
+struct spwd {
```

Comment:
This is kinda getting into an uncomfortable state of trying to cater to missing things on random platforms with ifdef soup...
Can we somehow...not?

Reply:
Well, the other option I see is to make the `spwd` arguments for `from_passwd` and `Account::Account` conditional, but that would mean even more `#ifdef`s.

Reply:
:( 
Maybe we could instead not build these bits on host, they're not used for anything anyway?

This PR is okay though.